toolbox

```diff
@@ -638,6 +744,42 @@ case $op in
         $ls_containers && list_containers
         exit
         ;;
+    rm | rmi )
```

Comment:
We need to plug in the `forward_to_host` thing that we added in commit 5b3d234c9e9ef45f.

Reply:
Done.

We should probably not touch the current container and it's image when the command is invoked from inside a toolbox. Let's address that in a later PR.